src/@next/components/molecules/CreditCardNumber/CreditCardNumber.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import { CreditCardIcon } from "../../atoms";
```

Comment:
Use absolute imports to reference components from other groups:
`import { CreditCardIcon } from "@components/atoms"`